### ISSUE TITLE
0.9-SNAPSHOT / master-branch NoSuchMethodError in splitAndFilterEmpty

### ISSUE THREAD

**Issue by imasen** [jenkinsci/docker-plugin#151]
I get the following runtime-exception if I try to save the docker-cloud configuration by splitAndFilterEmpty method:

```java
javax.servlet.ServletException: java.lang.NoSuchMethodError: com.google.common.base.Splitter.splitToList(Ljava/lang/CharSequence;)Ljava/util/List;
```

Java: Oracle Java 7 64-Bit
OS Jenkins-Master: OS X
OS Slave: CentOS 7
Docker Version: 1.2.4
Jenkins Version: 1.565.3

@carlossg You changed this method in PR 146, could you check this again. Did you use a newer version of Jenkins for testing?

**Comment by imasen** [comment]
After I changed back to the old-style method, saving is working again, without exception.

**Comment by KostyaSha** [comment]
Always provide full log + installed versions.

**Comment by carlossg** [comment]
Fixed in #156
